frontend/src/app/components/ExperimentPage.js

```diff
@@ -114,7 +114,7 @@ export default class ExperimentPage extends React.Component {
             thumbnail, survey_url, contributors, details, min_release } = experiment;
 
     const subtitle = (title === 'No More 404s') ? 'Powered by the Wayback Machine' : '';
-    const installation_count = experiment.installation_count.toLocaleString();
+    const installation_count = (experiment.installation_count) ? experiment.installation_count.toLocaleString() : '0';
```

Comment:
This is a hack. Probably needs something more elegant maybe in [`fetchExperiments.js`](https://github.com/mozilla/testpilot/blob/c401d646af6eb2102e1db4da7342572515638eca/frontend/src/app/actions/experiments.js#L16), suggestions welcome.